### ISSUE TITLE
tvOS - video sessions not reported as stopped

### ISSUE THREAD

**Issue by mueslimak3r** [jellyfin/Swiftfin#402]
### Describe the bug

After video playback is stopped, the jellyfin dashboard continues to show the video as playing.

To reproduce:
* start video playback and make sure the session shows up in the jellyfin dashboard
* end playback by pressing "menu" on the apple tv remote
* observe that the session remains in the jellyfin dashboard and continues to show playback advancing

Notes:
I recently got an m1 macbook so I may dig into the code and see if I can track this one down

### Application version

1.0.0 (74)

### Where did you install the app from?

TestFlight

### Device information

4th Gen Apple TV (A1625)

### OS version

tvOS 15.3 (19K547)

### Jellyfin server version

10.8-alpha5 (official docker image)

**Comment by LePips** [comment]
Yeah, this happens because the object that should send the stop report gets killed before it can.
